src/main/java/tasks/Event.java

```diff
@@ -0,0 +1,43 @@
+package tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    private String description;
+    private String time;
+
+    /**
+     * A constructor for the Event task.
+     *
+     * @param description user input task description.
+     * @param isDone status of the task.
+     */
+    public Event(String description, Boolean isDone) {
+        super(description, isDone, "E");
+        this.description = description;
+    }
+
+    /**
+     * Returns the status and description of the event task.
+     *
+     * @return a string representation of the event task.
+     */
+    public String getTask() {
+        // split text and duration
+        String[] splitted = description.split("/from ", 2);
+        String text = splitted[0].trim();
+        String duration = splitted[1].trim();
+
+        // split start date and end date
+        String[] splitted2 = duration.split("/to ", 2);
```

Comment:
Maybe you could change the variable names splittted and splitted2 to something more meaningful?
(Sorry that its a really small thing, this was like the only thing that i could find as a possible improvement)